### PR TITLE
Send locales to flutter includeing default locale

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bbrto21** [comment]
#36

**Comment by bbrto21** [comment]
**en_US**
image
**fr_FR**
image
**es_ES**
image

**Review by xiaowei-guan — APPROVED** [review]
LGTM

File: shell/platform/tizen/channels/localization_channel.cc [hunk 1]
```diff
@@ -21,30 +21,50 @@ LocalizationChannel::LocalizationChannel(FLUTTER_API_SYMBOL(FlutterEngine)
...
+  without_encoding_type.erase(n, without_encoding_type.length() - n);
+
+  flutter_locale = GetFlutterLocale(without_encoding_type.data());
+  if (flutter_locale) {
+    FT_LOGD("Choose Default locale[%s]", without_encoding_type.data());
+    flutter_locales.push_back(flutter_locale);
+  }
+
   int32_t count = i18n_ulocale_count_available();
   if (count > 0) {
```
Comment (swift-kim) [inline shell/platform/tizen/channels/localization_channel.cc:46]: Don't you need to remove this condition?

**Review by bwikbs — APPROVED** [review]

File: shell/platform/tizen/channels/localization_channel.cc (hunk 1 above)
Comment (bbrto21) [reply shell/platform/tizen/channels/localization_channel.cc]: I fixed it

**Pull request merged**